README.md

```diff
@@ -37,124 +53,382 @@ To install the global command line interface (CLI), use npm global installation:
-| :--------------------- | :-----: | :-------: | :--------------------------------------------------------------------------------- |
-| verboseCalculationResults | boolean | yes | Use the detailed results interfaces for calculation. Defaults to true. |
-| enableDebugOutput | boolean | yes | Enable debug output from function calls. Defaults to false. |
-| includeClauseResults | boolean | yes | Option to include clause results. Defaults to false. |
-| measurementPeriodStart | string | yes | Start of measurement period. |
-| measurementPeriodEnd | string | yes | End of measurement period. |
-| patientSource | DataProvider | yes | PatientSource to use. If provided, the patientBundles are not required. The PatientSource/DataProvider interface is defined in TypeScript by [cql-execution](https://github.com/cqframework/cql-execution/blob/master/src/types/cql-patient.interfaces.ts). |
-| reportType | string | yes | Type of MeasureReport to generate: "summary", "subject-list" (not yet supported), or "individual". |
-| calculateSDEs | boolean | yes | Include Supplemental Data Elements in calculation. Defaults to true. |
-| calculateHTML | boolean | yes | Include HTML structure for highlighting. Defaults to true. |
-| calculateClauseCoverage | boolean | yes | Include HTML structure with coverage highlighting. Defaults to true. |
-| vsAPIKey | string | yes | API key, to be used to access a ValueSet API for downloading any missing ValueSets |
-| useValueSetCaching | boolean | yes | Whether to cache ValueSets obtained by an API on the filesystem. Defaults to false. |
-| useElmJsonsCaching | boolean | yes | Whether to cache the ELM JSONs and associated information from calculation. Defaults to false. |
-| clearElmJsonsCache | boolean | yes | Whether to clear the ELM JSONs cache before running calculation. Defaults to false. |
-| trustMetaProfile | boolean | yes | To "trust" the content of meta.profile as a source of truth for what profiles the data that [cql-exec-fhir](https://github.com/projecttacoma/cql-exec-fhir) grabs validates against. Defaults to false.|
-
-### CLI
-
-To run the globally installed CLI (see above), use the global `fqm-execution command`
 
-```bash
+- `[calculateClauseCoverage]`<[boolean]()>: Include HTML structure with clause coverage highlighting (default: `true`)
+- `[calculateHTML]`<[boolean]()>: Include HTML structure for highlighting (default: `true`)
+- `[calculateSDEs]`<[boolean]()>: Include Supplemental Data Elements (SDEs) in calculation (default: `true`)
+- `[clearElmJsonsCache]`<[boolean]()>: if `true`, clears ELM JSON cache before running calculation (default: `false`)
+- `[enableDebugOutput]`<[boolean]()>: Enable debug output including CQL, ELM, results (default: `false`)
+- `[includeClauseResults]` <[boolean]()>: Option to include clause results (default: `false`)
+- `[measurementPeriodEnd]`<[string]()>: End of measurement period
+- `[measurementPeriodStart]`<[string]()>: Start of measurement period
```

Comment:
Do we want to add a little blurb here about how these being optional works?

Reply:
yes, good call